src/validate.js

```diff
@@ -179,7 +180,7 @@ function validateField(formField, model, value) {
 
     // update error
     utils.selectOrSet(
-      getFieldKey(formField),
+      getFieldKey(formField)[0], // for nested props. e.g. if type is array of autocmpletes
```

Comment:
This change is not correct. schema-form supports nested models with references such as address.street that become ['address', 'street'] array keys and the complete array is necessary to update the model.